tests/experimental/pydantic/test_basic.py

```diff
@@ -63,6 +64,73 @@ class UserType:
     assert field2.type.of_type is str
 
 
+def test_basic_type_with_input_metadata():
+    class User1(pydantic.BaseModel):
+        friend: Optional[str] = "friend_value"
+        frenemy: str
+
+    @strawberry.experimental.pydantic.type(User1, with_input_metadata=True)
+    class UserType1:
+        friend: strawberry.auto
+        frenemy: strawberry.auto
+
+    user_input_1 = UserType1(frenemy="frenemy_value")
+    assert is_unset(user_input_1.friend)
+    assert user_input_1.to_pydantic().friend == "friend_value"
```

Comment:
add tests for  defaults converted from pydantic
```suggestion
    assert user_input_1.friend == "friend_value"
    assert user_input_1.to_pydantic().friend == "friend_value"
```